board/fischertechnik/TXT/device_table_initramfs.txt

```diff
@@ -0,0 +1,11 @@
+# See package/makedevs/README for details
```

Comment:
I guess this is the migration of initramfs.conf.

Reply:
more or less. this just defines directorys to be created, permissions and co. Just like the board/fischertechnik/TXT/device_table_rootfs.txt